.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,54 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  test36:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.6]
```

Comment:
do we still need 3.6 with it being EOL and basically phased out of almost everything?

Reply:
Yes, 3.6 is the main python in sles15 in which the adapter will run against. Unless we choose to use a none default interpreter. But afaik we plan to use the default.

Reply:
Agreed; so long as the default SLE python version is 3.6, we are stuck with 3.6 testing... Once they get the Python 3.11 support into the SLE Python 3 module we might be able to switch to requiring it, and dropping the need to be Python 3.6 compatible...